Owncloud iOs Client/Login/Login/UniversalLoginViewController.swift

```diff
@@ -702,21 +903,22 @@ connection_declined  Connection declined by user
                                                 
                                             } else {
                                                 
+                                                self.setNetworkActivityIndicator(status: false)
+                                                
                                                 //TODO: check with https url without prefix, error unsupported URL
-                                                //                                                if errorHttp == 0 {
-                                                    //                                                    self.setPasswordFooterError(message: NSLocalizedString("", comment: "") )
-                                                    //
-                                                    //                                                } else {
+                                                if errorHttp == 0 {
```

Comment:
Not yet control this todo, leave it commented, is needed to review the error handling for some kind of errors